Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityFocusProvider.cs

```diff
@@ -54,6 +54,15 @@ public interface IMixedRealityFocusProvider : IMixedRealityService, IMixedRealit
         /// <param name="focusDetails"></param>
         bool TryGetFocusDetails(IMixedRealityPointer pointer, out FocusDetails focusDetails);
 
+        /// <summary>
+        /// Sets the FocusDetails of the specified pointer, overriding the focus point that was currently set. This can be used to change
+        /// the FocusDetails of a specific pointer even if focus is locked.
```

Comment:
Could you provide an example of why / when you might want to do this (for example when panning content)?